### DISCUSSION TITLE
Hyundai bluelink in home assistant

### DISCUSSION THREAD

**Discussion by pbHub01** [Hyundai-Kia-Connect/kia_uvo#850] [Q&A] [answered]
Hi,
I'm new here. Can you please help me with Hyundai bluelink api in Home assistant. I'm in Canada. I was able to paste the custom repository and when I go to integration, I only see USA region and Kia brand. If I go with that, it doesn't let me access my Hyundai account.
Here are the things I see:
![image](https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/assets/162990879/5dd67479-4930-4f3e-8585-5f1981d2ed47)

![image](https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/assets/162990879/ee2cebc1-3ae9-43fe-a6dd-017e97fbc0d2)

Thanks you

**Comment by cdnninja** [comment, marked as answer]
That doesn't look like Kia _uvo but rather another integration that only supports USA.

**Reply by pbHub01** [reply]
Yes you are right. My mistake, I didn't get it directly from GitHub, so after reading your reply I integrated the one I found on GitHub and it works now. I've learned my lesson, it's better to search directly in GitHub. Thank you for your reply.
